website/static/js/fangorn.js

```diff
@@ -1826,6 +1794,30 @@ function _dragLogic(event, items, ui) {
     return copyMode;
 
 }
+
+function getCopyMode(folder, items) {
+    var tb = this;
+    var canMove = true;
+
+    if (folder.parentId === 0) return 'forbidden';
+    if (folder.data.kind !== 'folder' || !folder.data.permissions.edit) return 'forbidden';
+    if (!folder.data.provider || folder.data.status) return 'forbidden';
+
+    for(var i = 0; i < items.length; i++) {
+        var item = items[i];
+        if (item.data.nodeType) return 'forbidden';
```

Comment:
Mostly because its easier to reason about, at least I think so.
Happy to change it if you'd like me to. :+1: